File: packages/backend/src/main/kotlin/hollybike/api/types/event/participation/EEventRole.kt
/*
  Hollybike API Kotlin KTor Graalvm application
  Made by MacaronFR (Denis TURBIEZ) and Loïc Vanden Bossche
*/
package hollybike.api.types.event.participation

enum class EEventRole(val value: Int) {
	Member(1),
	Organizer(2);

	companion object {
		operator fun get(value: Int) = EEventRole.entries.first { it.value == value }
	}
}

File: packages/frontend/src/types/EEventRole.ts
/*
  Hollybike Back-office web application
  Made by MacaronFR (Denis TURBIEZ) and enzoSoa (Enzo SOARES)
*/
import { Option } from "../components/Select/Select.tsx";

export enum EEventRole {
	Member = "Member",
	Organizer = "Organizer"
}

export const EEventRoleOptions: Option[] = [
	{
		name: "Membre",
		value: EEventRole.Member,
	},
	{
		name: "Organisateur",
		value: EEventRole.Organizer,
	},
];
